.dev/cc.R

```diff
@@ -70,6 +70,17 @@ cc = function(test=FALSE, clean=FALSE, debug=FALSE, omp=!debug, path=Sys.getenv(
   dll = grep("data_table.so", dll, fixed=TRUE, value=TRUE)
   sapply(dll, dyn.unload)
   gc()
+  
+  # hack for windows under mingw-like environment
+  # PROJ_PATH must be windows like i.e.: export PROJ_PATH=$(Rscript -e 'getwd()')
+  if (.Platform$OS.type == "windows") {
+    system = function(command, ...) shell(shQuote(command, "sh"), shell = "sh",...)
+    # when -std=c99, macro WIN32 is not defined (MinGW) so we define it
+    # otherwise will fail as sys/mman.h does not exists in windows (fread.c)
+    W32 = "-DWIN32"
+  } else {
+    W32 = ""
```

Comment:
```suggestion
    W32 = "-DWIN32"
    dt_object = "data_table"
  } else {
    W32 = ""
    dt_object = "data_table.so"
```

And then return it below rather than branch again